### ISSUE TITLE
Localstack doesn't return some metadata for S3 Range requests

### ISSUE THREAD

**Issue by lukaszdutka-vonage** [localstack/localstack#1859]
Using s3 range request with localstack causes incorrect content type to be returned & some metadata to be missing:
To reproduce, put object:
`aws s3api put-object --endpoint http://localhost:3006 --bucket some-bucket --key some-dir/some-key --body text.txt --content-type text/plain --content-language en-US --cache-control no-cache`

And compare regular request vs range one:
`aws s3api get-object --endpoint http://localhost:3006 --bucket some-bucket --key some-dir/some-key output.txt --range bytes=0-3`
`aws s3api get-object --endpoint http://localhost:3006 --bucket some-bucket --key some-dir/some-key output.txt`

This works correctly (returns the same metadata regardless of using `--range bytes=0-3`) on aws s3.

Also:
Getting object with/without `range bytes=0-3` should return header "AcceptRanges: bytes" but it doesn't.

**Comment by whummer** [comment]
Thanks for reporting @lukaszdutka-vonage . This should be fixed in #2326 - can you please give it a try with the `latest` version of the Docker image? Please report here if the problem persists. Thanks

**Comment by lukaszdutka-vonage** [comment]
Hello whummer! It's been a while since creating this issue 😅 I tested it today and my problem persists. I checked the code that were merged in #2326 and I have to say that it's not everything that we needed in our project :(

By "metadata" we meant fields like (in our case):
"content-language" with value "en-US",
"cache-control" with value "no-cache" and
"content-encoding" with value "identity"

and those are still not present in the response metadata.
